types/json8-patch/index.d.ts

```diff
@@ -60,7 +61,7 @@ export type JsonPatchOperation = AddOperation | RemoveOperation | ReplaceOperati
 export type JsonPatch = JsonPatchOperation[];
 
 export interface ApplyResult {
-    doc: any;
+    doc: unknown;
```

Comment:
Why `unknown` here as opposed to `any`? It seems like `doc` is any in a lot of places in this package.